tensorflow/python/framework/tensor_util_test.py

```diff
@@ -19,6 +19,7 @@
 from __future__ import print_function
 
 import numpy as np
+import pandas as pd
```

Comment:
We can't have unguarded pandas imports -- you can try to import it -- see pandas imports in contrib/learn